packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1884,19 +1919,34 @@ class ThemeData with Diagnosticable {
     Color? selectedRowColor,
     @Deprecated(
       'Use colorScheme.error instead. '
-      'This feature was deprecated after v2.6.0-11.0.pre.',
+      'This feature was deprecated after v3.3.0-0.5.pre.',
     )
     Color? errorColor,
     @Deprecated(
       'Use colorScheme.background instead. '
-      'This feature was deprecated after v2.6.0-11.0.pre.',
+      'This feature was deprecated after v3.3.0-0.5.pre.',
```

Comment:
These also appear unrelated. Why is this version changing?

Reply:
This fixes a copy-paste error from a past PR

Reply:
Can you link the PR?

Reply:
https://github.com/flutter/flutter/pull/110162, linked in description